addons/web/static/src/views/form/form_controller.js

```diff
@@ -188,6 +188,7 @@ export class FormController extends Component {
             getLocalState: () => {
                 // TODO: export the whole model?
                 return {
+                    activePages: !this.model.root.isNew && this.activePages,
```

Comment:
maybe rename `activeNotebookPages` (everywhere where it's not clear from context that we talking about notebook)

Reply:
I'm afraid this isn't where we should check about the isNew flag (or not only, at least). Here you do not export the state when you **leave** a new record. But what you really don't want is not restore the state when you **enter** a new record.

For for instance, in a list, click on an existing record, click second notebook tab, go back, click create -> you are on the second tab, right?